firefox-ios/Client/Frontend/Fakespot/Views/FakespotMessageCardButton.swift

```diff
@@ -0,0 +1,103 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import ComponentLibrary
+
+public struct FakespotMessageCardButtonViewModel {
+    let title: String
+    let a11yIdentifier: String
+    let type: FakespotMessageCardViewModel.CardType
+
+    init(title: String = "",
+         a11yIdentifier: String = "",
```

Comment:
Do we want empty defaults on title and identifiers? Shouldn't the buttons always have those?

Reply:
The button doesn't exist for all the different card types. And depending on the card the a11y id is different. So the  alternative would be to create an a11y that would never be used as a default which I don't like very much.